### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Added logo to the E commerce site
*Linked the pages to each other through i tag
*Added log out page https://deploy-preview-6--lucidbook-store.netlify.app/pages/logout.html

### CODE REVIEW

**Review by GuptaEesh — APPROVED** [review]
Code looks crisp and great use of variables. Good to go.

File: css/homepage.css [hunk 1]
```diff
@@ -34,12 +34,15 @@
...
-  border: 9px solid var(--blue);
+  border: 9px solid var(--very-light-blue);
   border-radius: 50px;
   width: 100%;
   overflow: hidden;
   height: 45rem;
 }
+.hero-text{
+  
+}
```
Comment (Hamdrive) [inline css/homepage.css]: If there is not properties defined then this can be cleaned.

File: index.html [hunk 2]
```diff
@@ -12,10 +12,14 @@
...
       <!-- navbar -->
-      <nav class="navbar shadow">
+      <nav class="navbar li-shadow">
         <div class="nav-brand">
           <a href="" class="logo-a">
-            <i class="fas fa-suitcase"></i>
+            <img
+              class="logo-s"
+              src="images/02 - Colorful - Icon Only.png"
+              alt=""
```
Comment (Hamdrive) [inline index.html]: consider giving alt value, as it's logo you can give `alt="Colorful logo"`

File: index.html [hunk 3]
```diff
@@ -24,28 +28,35 @@
...
-            <a href="pages/cart.html" class=""><button class="btn-icon nav-icons m-x-1">
-              <i class="fas is-dark fa-shopping-cart"></i>
-            </button></a>
-            <button class="btn-icon nav-icons m-x-1">
-              <i class="fas fa-bell"></i>
-            </button>
+            <a href="pages/cart.html" class=""
+              ><button class="btn-icon nav-icons m-x-1">
+                <i class="fas is-dark fa-shopping-cart"></i></button
+            ></a>
```
Comment (Hamdrive) [inline index.html:44]: Refactor prettifier for better code formatting

File: index.html [hunk 4]
```diff
@@ -81,60 +84,57 @@
...
           </div>
           <div class="scale-1 category-product m-l-1 dk-shadow">
-            <img
-              src="images/emotions.png"
-              alt=""
-              class="category-img"
-            />
-            <p class="is-3 m-y-1 m-l-1 semibold category-name is-">trader psychology</p>
+            <img src="images/emotions.png" alt="" class="category-img" />
+            <p class="is-3 m-y-1 m-l-1 semibold category-name is-">
```
Comment (Hamdrive) [inline index.html]: is the last utility class a typo? or is it actually called `is-` ?

File: index.html [hunk 5]
```diff
@@ -174,66 +175,73 @@
...
+            </div>
+            <div class="card-body">
+              <div class="textbox">
+                <div class="title">Learn from Professional</div>
+                <!-- <div class="subtitle">Subtitle</div> -->
               </div>
-              <div class="card-body">
-                <div class="textbox">
-                  <div class="title">Learn from Professional</div>
-                  <!-- <div class="subtitle">Subtitle</div> -->
```
Comment (Hamdrive) [inline index.html:194]: Cleanup unwanted code comments

File: pages/cart.html [hunk 6]
```diff
@@ -13,35 +13,41 @@
...
-          <button class="btn-icon nav-icons m-x-1">
-            <i class="fas fa-heart"></i>
-          </button>
-          <button class="btn-secondary nav-btn btn-small">Sign up</button>
-          <button class="btn-primary nav-btn btn-small">Log in</button>
+          <a href="../pages/cart.html" class=""
+            ><button class="btn-icon nav-icons m-x-1">
+              <i class="fas is-dark fa-shopping-cart"></i></button
+          ></a>
+          <a href="../pages/wishlist.html" class="">
```
Comment (Hamdrive) [inline pages/cart.html]: cleanup empty class attribute if not used

File: pages/login.html [hunk 7]
```diff
@@ -5,80 +5,89 @@
...
-                <a href="" class="is- link">Create Account</a>
-            </div>
+      <div class="login b-radius center-x m-up-6 shadow p-y-2 p-x-4">
+        <div class="textbox">
+          <div class="title">Login</div>
+        </div>
+        <div class="form-div m-up-1">
+          <p class="form-label">Email</p>
+          <input
+            type="text"
```
Comment (Hamdrive) [inline pages/login.html]: use the `type=email` attribute for email input

File: pages/login.html [hunk 8]
```diff
@@ -5,80 +5,89 @@
...
+          <p class="form-label m-up-2">Password</p>
+          <input
+            type="password"
+            class="form-input input-focused"
+            placeholder="enter your password"
+            required
+          />
+        </div>
+        <label for="" class="m-up-2 form-checkbox">
+          <input type="checkbox" class="" />Remember me
```
Comment (Hamdrive) [inline pages/login.html]: might not need `for` attribute if input is nested within label, else you can link `for` attribute to `id` attribute of input

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Great work done, good naming conventions for class names.

Request the following suggestions to be looked into:
1. use `alt` tag where needed.
2. better code formatting
3. clean unnecessary code comments
4. other comments mentioned in review

File: css/homepage.css (hunk 1 above)
Comment (AnuragSahu11) [reply css/homepage.css]: removed

File: index.html (hunk 2 above)
Comment (AnuragSahu11) [reply index.html]: Yes will keep in mind from now on

File: index.html (hunk 4 above)
Comment (AnuragSahu11) [reply index.html]: it was a typo fixed it thanks for pointing out

File: index.html (hunk 5 above)
Comment (AnuragSahu11) [reply index.html:194]: removed

File: pages/cart.html (hunk 6 above)
Comment (AnuragSahu11) [reply pages/cart.html]: Yup

File: pages/login.html (hunk 7 above)
Comment (AnuragSahu11) [reply pages/login.html]: changed

File: index.html (hunk 3 above)
Comment (AnuragSahu11) [reply index.html:44]: I will see to that

**Review by Hamdrive — APPROVED** [review]
Great work implementing the changes given
Code looks good and ready to push!

**Pull request merged**